docs/9-membership.md

```diff
@@ -143,134 +157,148 @@ Read more about how weighting is determined in [**6.3 Weighting**](https://proto
+ | 0x661b81d462D80786c442774F452464A8C627a20E | 0.6207 |
+ | 0x2fb858991668840ce34F331402E0b3C66db078AF | 0.6207 |
+ | 0xB7b93e217dd4E6E700E7362cB234d6258438D3b7 | 0.6207 |
+ | 0x78ac9c2545850bEDbC076EB30ce7A6f0D74b395E | 0.6341 |
+ | 0x2D56Cd519540bE541A3261E22e95d6507F5504Ca | 0.6341 |
+ | 0x3210287ddEe6ece40d17A1F39799239b972A81F2 | 0.6341 |
+ | 0x153afFb96Fcb60085Ee307996Bdd2df0183A3682 | 0.6341 |
+ | 0x71c15691e243bE88220957C784053EF0E084440B | 0.6341 |
+ | 0x980a85ba6c2683e3509752dd3b4eB50165C0e65F | 0.6341 |
+ | 0xa1506f8DC05b1c22Ba23CBfd4625C68599196E91 | 0.6341 |
+ | 0x5abAdE91eD4B6d11b666280CaCb3E4A32898f39B | 0.6407 |
+ | 0x0760E844e6f368ce73F1eEB917d37Db19375De3B | 0.6472 |
+ | 0x50122A5509F628e901F9c0238F0168833753239b | 0.6472 |
+ | 0xf0443945aD3BE9645382FC2537317dA97FEfF3A9 | 0.6472 |
+ | 0x10c8597a5063A1648FfE13f54E996ba9bB3217B5 | 0.6472 |
+ | 0x0B916095200313900104bAcfc288462682C38700 | 0.66 |
+ | 0x2bf7b04F143602692bBDc3EcbeA68C2c65278eee | 0.6725 |
+ | 0x84f678A3e7BA8Fc817c32Ff10884D6FB20976114 | 0.6725 |
+ | 0x5AC41B7E73680a3E77B941fEE0BAD04F59c9bB7d | 0.6725 |
+ | 0xCb8DC3beC7B659022aE0d3E9de17322F31e4AA7C | 0.6849 |
+ | 0x86F34D8b98171281AB8bFe65C7e2718E4f002e35 | 0.691 |
+ | 0x4d5083dd10f2a46f26f5583c6679f9f8d30ae850 | 0.697 |
+ | 0xe611a720778a5f6723d6b4866F84828504657181 | 0.697 |
+ | 0xAb96014a7c078f09418Cf899Bf197CadFf023C16 | 0.7089 |
+ | 0xC152fd31F285f6c0B3807070280595e7Ea713a7f | 0.7089 |
+ | 0xbC349D1BEeE33c61F0395d1667E70056B4C869B9 | 0.7206 |
+ | 0x644177F8D79117c2b9C7596527642B3c2D05888E | 0.7322 |
+ | 0x3eD7bf997b7A91e9e8aB9eE2F7ce983bd37D6392 | 0.7379 |
+ | 0xFf9977FB117a22254a8eB6c6CE8d3Dd671FA70DC | 0.7379 |
+ | 0xf71E9C766Cdf169eDFbE2749490943C1DC6b8A55 | 0.7379 |
+ | 0xb721c2e6640D963e99b37B6437ABAF6914A25A5e | 0.7435 |
+ | 0x9915C453cc779109013F1aF0c4639277e8257212 | 0.7435 |
+ | 0xa1D76553266fA8Ed3D143794a462aaFAdfC34f74 | 0.7547 |
+ | 0xBc11295936Aa79d594139de1B2e12629414F3BDB | 0.7547 |
+ | 0x49Aa00072a7BA8a5639C43Fe9F6536820E6F5d57 | 0.7657 |
+ | 0xC6cA7c3427AD6B7a06fbED6D18C394E540E31814 | 0.7873 |
+ | 0x46cD90445349e64F895c403c23839e79eb4065e4 | 0.7979 |
+ | 0xD55Bf29915D139777783bBA3Ae6237844823B73b | 0.8186 |
+ | 0x88dF07c605d13915B8E7D07dc8efD538dC0C5620 | 0.8186 |
+ | 0xE4b99f9580B90d88C3e6dC3E0B853D8D3b0B8B55 | 0.8186 |
+ | 0x92699d64C65c435D4a60E2ceEaEb931dB8B1cA09 | 0.8288 |
+ | 0xa29576F07eAaD050AeACbc89AC0518B62fe2F88E | 0.8388 |
+ | 0x80Bb92f389591d353654ac5580baefF7d0d6e778 | 0.8388 |
+ | 0xe05875F287C028901798aC2Dc8C22Ba908b8eF36 | 0.8487 |
+ | 0x004f67dabb603aaa58ed52641ccaff09c559704a | 0.8586 |
+ | 0x799681f9b9d7c55ed59f5a0f235cAb132Cde0a2B | 0.8586 |
+ | 0xb7A593EC62dc447eef23ea0e0B4d5144ac75ABC5 | 0.8586 |
+ | 0xD4a3030b5f5e8DD4860d370C17E8576aE9951a2D | 0.8778 |
+ | 0x10ab73AA48D686b7FD9ec9D50418a14DD23f6631 | 0.8778 |
+ | 0x6e22a5e30088c8389dc725bbead5f0675334299f | 0.8873 |
+ | 0x24113fFB07189D1e6E169025A424B58C29522972 | 0.9152 |
```

Comment:
Yep 0xSplits is out of date, since it's a snapshot of the weights at the time the last membership update was made onchain. 

Whereas my changes reflect what we have in the Airtable, which is automatically calculated based on today's date! So it's technically more accurate, but  I dont think that's worth the potential confusion. So I'll edit to use 0xSplits as the source instead of Airtable. Thanks Justin!

Reply:
Okay cool, thanks 😄